CODE_OF_CONDUCT.md

```diff
@@ -168,41 +203,43 @@ Our ability to address any code of conduct breaches in a timely and effective ma
 - Names and contact info, if possible, of **anyone else who witnessed** or was involved in this incident. (Did anyone else observe the incident?)
 - **Any other relevant information** you believe we should have
 
-At conda Organization Events: Event staff will attempt to gather and write down the above information from anyone making a verbal report in-person at an event. Recording the details in writing is exceedingly important in order for us to effectively respond to reports. If event staff write down a report taken verbally, then the person making the report will be asked to review the written report for accuracy.
+## Anonymous Reporting
+
+The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, than the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.
```

Comment:
```suggestion
The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, then the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.
```